src/components/ens-registration/IntroMarquee/IntroMarquee.tsx

```diff
@@ -55,15 +55,15 @@ export default function IntroMarquee({
     []
   );
 
-  const items = useMemo(
-    () =>
-      ensIntroMarqueeNames.map((name, index) => ({
-        name,
-        onPress: () => handlePressENS(name),
-        testID: `ens-names-marquee-item-${index}`,
-      })),
-    [handlePressENS]
-  );
+  const items = useMemo(() => {
+    return isLoading
+      ? []
+      : data?.ensMarquee?.accounts?.map((account, index) => ({
+          account,
+          onPress: () => handlePressENS(account.name),
+          testID: `ens-names-marquee-item-${index}`,
+        }));
+  }, [data, handlePressENS, isLoading]);
```

Comment:
What happens if it's done loading but there's an `error` and no `data`? Do we have any fallback?